spec/grape/validations/validators/coerce_spec.rb

```diff
@@ -752,19 +776,6 @@ def self.parsed?(value)
         expect(last_response.body).to eq('String')
       end
 
-      it 'respects nil values' do
```

Comment:
This is no longer working?

Reply:
this spec is actually the opposite and yes it's breaking #2019. But #2019 is why we have this conversation :)

Reply:
I might be flip-flopping on what we want, let's step back. For this example alone, when I say `:a` can be either an array of `File` or a `String`, and I pass `nil`, don't I *want* a `nil` and *not* `[]`?

Reply:
> For this example alone, when I say :a can be either an array of File or a String, and I pass nil, don't I want a nil and not []?

That was the behavior of Grape prior to v1.3.0. v1.3.x broke our tests with that, which is why I filedhttps://github.com/ruby-grape/grape/issues/2018.